CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## [Unreleased]
 
+## [2.0.0] - 2017-05-02
+### Added
+- Added new `tslint` rules from [v5.0.0](https://github.com/palantir/tslint/blob/master/CHANGELOG.md#v500) and [v5.1.0](https://github.com/palantir/tslint/blob/master/CHANGELOG.md#v510) releases
```

Comment:
I don't understand your question.  To be clear, I'm saying:
> This list makes it look like we've enabled 7 new rules, but we've really enabled 5

i.e., should the list provide more context like:

```
* `no-invalid-template-strings`
* `no-sparse-array` (disabled)
```

... or are we happy with consumers digging in to see what is/isn't enforced by default?